records_mover/db/postgres/loader.py

```diff
@@ -0,0 +1,133 @@
+    def load(self,
+             schema: str,
+             table: str,
+             load_plan: RecordsLoadPlan,
+             directory: RecordsDirectory) -> None:
+        all_urls = directory.manifest_entry_urls()
+
+        for url in all_urls:
+            loc = self.url_resolver.file_url(url)
+            with loc.open() as f:
+                # Postgres COPY FROM defaults to appending data--we
+                # let the records Prep class decide what to do about
+                # the existing table, so it's safe to call this
+                # multiple times and append until done:
+                logger.info(f"Loading {url} into {schema}.{table}")
+                self.load_from_fileobj(schema, table, load_plan, f)
+
+    def can_load_this_format(self, source_records_format: BaseRecordsFormat) -> bool:
+        try:
+            processing_instructions = ProcessingInstructions()
+            load_plan = RecordsLoadPlan(records_format=source_records_format,
+                                        processing_instructions=processing_instructions)
+            if not isinstance(load_plan.records_format, DelimitedRecordsFormat):
+                return False
+
+            unhandled_hints = set(load_plan.records_format.hints.keys())
+            processing_instructions = load_plan.processing_instructions
+            postgres_copy_options(unhandled_hints, load_plan)
+            complain_on_unhandled_hints(processing_instructions.fail_if_dont_understand,
+                                        unhandled_hints, load_plan.records_format.hints)
+            return True
+        except NotImplementedError:
+            return False
+
+    def known_supported_records_formats_for_load(self) -> List[BaseRecordsFormat]:
+        return [
+            # To validate that these load without, watch logging while
+            # running and verify 'dataframe' doesn't appear:
+            #
+            # ./itest shell
+            #
+            # mvrec file2table --source.variant bluelabs
+            # --source.no_compression
+            # tests/integration/resources/delimited-bluelabs-no-header.csv
+            # dockerized-postgres public bluelabsformat
+            DelimitedRecordsFormat(variant='bluelabs',
+                                   hints={'compression': None}),
+            # mvrec file2table --source.variant csv
+            # --source.no_compression
+            # tests/integration/resources/delimited-csv-with-header.csv
+            # dockerized-postgres public csvformat
+            DelimitedRecordsFormat(variant='csv',
+                                   hints={'compression': None}),
+            # mvrec file2table --source.variant bigquery
+            # --source.no_compression
+            # tests/integration/resources/delimited-bigquery-with-header.csv
+            # dockerized-postgres public bigqueryformat
+            DelimitedRecordsFormat(variant='bigquery',
+                                   hints={'compression': None}),
```

Comment:
Postgres doesn't support loading compressed files.  Maybe in the future we can teach records mover to compress/decompress as needed for databases without going all the way to a dataframe and back when that's the only missing piece...